activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-jpa/src/main/java/org/activiti/cloud/services/modeling/jpa/ModelRepositoryImpl.java

```diff
@@ -64,6 +64,17 @@ public ModelRepositoryImpl(ModelJpaRepository modelJpaRepository){
         }
     }
 
+    @Override
+    public Optional<ModelEntity> findGlobalModelByNameAndType(String modelName, String modelTypeFilter) {
+        List<ModelEntity> models = modelJpaRepository.findModelByNameAndScopeAndTypeEquals(modelName, ModelScope.GLOBAL, modelTypeFilter);
+
+        if (models != null && !models.isEmpty()) {
+            return Optional.of(models.get(0));
```

Comment:
I assume that this query cannot return more than one result, right?

Reply:
Yes that is.